### ISSUE TITLE
A+ should show a clear error if the submission fails due to large files

### ISSUE THREAD

**Issue by Mankro** [apluslms/a-plus#869]
The user should see a clear error if the submission fails due to attempting to submit too large files. Currently, when this occurs in a chapter exercise, the modal dialog shows a generic error message ("There was a problem loading the assignment. Try loading the page again later. The course staff has been notified. If the problem persists for over a day and hasn't been addressed on the course news, please contact course staff.").

The file upload may fail in the Kubernetes ingress or Nginx HTTP server, which means that the request does not always reach the A-plus Django code. Therefore, the problem can be detected in the client (browser) from the HTTP response code of the submission POST request. Response HTTP 413 Payload too large indicates this problem.

This is easy to do in chapter.js for chapter exercises. Full-view exercises (outside chapters) are more complicated since the submission is not uploaded in JavaScript code, though perhaps the browser then shows a full 413 error page. Let's keep this simple and fix at least chapter.js. Full-view exercises can be tested to see if there is anything to do with them.

Chapter.js would be changed around this line:
https://github.com/apluslms/a-plus/blob/4271e485c768172a5317582d8ba3c73b04e006e4/exercise/static/exercise/chapter.js#L412

**Comment by Mankro** [comment]
Another complaint about the misleading error message (internal ticket):
https://rt.cs.aalto.fi/Ticket/Display.html?id=20170

**Comment by Mankro** [comment]
Related to "Configurable max file size for submissions" https://github.com/apluslms/a-plus/issues/662

**Comment by Mankro** [comment]
This could be one of Joni's first issues.
